http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeServiceInterp.scala

```diff
@@ -250,13 +250,11 @@ class RuntimeServiceInterp[F[_]: Parallel](
       samResources <- samService.listResources(userInfo.accessToken.token, RuntimeSamResource.resourceType)
 
       (labelMap, _, _) <- F.fromEither(processListParameters(params))
-      excludeStatuses = List(RuntimeStatus.Deleted)
       creatorOnly <- F.fromEither(processCreatorOnlyParameter(userInfo.userEmail, params, ctx.traceId))
 
       _ <- ctx.span.traverse(s => F.delay(s.addAnnotation("Start DB query for listRuntimes")))
       runtimes <- RuntimeServiceDbQueries
         .listRuntimes(samResources.map(RuntimeSamResourceId).toSet,
-                      excludeStatuses = excludeStatuses,
```

Comment:
I commented this in Slack, but figured I'd put this question in the PR: where else is this used? Will this start showing  deleted runtimes in Terra UI or from public-facing APIs?